frontend/package.json

```diff
@@ -35,7 +35,7 @@
     "react-leaflet": "^2.2.0",
     "react-router": "^4.3.1",
     "react-router-dom": "^4.3.1",
-    "react-scripts": "2.1.3",
+    "react-scripts": "^2.1.3",
```

Comment:
There's an unplanned change here as well as in the package-lock, can you please revert it?